firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/FunctionDeclarations.kt

```diff
@@ -213,7 +222,7 @@ class Schema<T>(
       )
 
     /** Registers a schema for a floating point number */
-    fun num(name: String, description: String) =
+    fun double(name: String, description: String) =
```

Comment:
We are trying to be more mindful of breaking changes in vertex. Could you mark the `num` version as deprecated, and add the `double` option as an additional one?